.teamcity/delivery/BuildAndPushHosted.kts

```diff
@@ -0,0 +1,47 @@
+package delivery
+
+import common.TeamCityDockerImagesRepo.TeamCityDockerImagesRepo
+import utils.ImageInfoRepository
+import utils.dsl.general.teamCityBuildDistDocker
+import utils.dsl.general.teamCityImageBuildFeatures
+import utils.dsl.steps.buildAndPushToStaging
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.dockerCommand
+
+/**
+ * Builds Ubuntu-based Server image and publishes it to staging.
+ * TODO: Remove configuration, as it duplicates the build included into PushLocalLinux2004.kts, while seems redundant.
+ */
+object BuildAndPushHosted : BuildType({
```

Comment:
Same as `publish_hub_latest` - I've left `TODO` in order to double-check if this configuration had been separated on purpose. 
In my assumption, it could be removed, as it only build `teamcity-server` and pushes it into staging registry, which is already implemented in context of `push_local_linux_20_04`.